src/ErrorHandler.php

```diff
@@ -106,22 +106,31 @@ public function memoryReserveSize(int $size): void
     }
 
     /**
-     * Register this error handler.
+     * Register PHP exception and error handlers and enable this error handler.
      */
     public function register(): void
     {
-        // Disables the display of error.
-        if (function_exists('ini_set')) {
-            ini_set('display_errors', '0');
+        if ($this->memoryReserveSize > 0) {
```

Comment:
```suggestion
        if ($this->memoryReserveSize > 0 && $this->memoryReserve === '') {
```

Reply:
Maybe add 

```
if ($this->enabled) {
    return;
}
```

to the beginning instead it?

Reply:
Yes, maybe.

Reply:
done